### ISSUE TITLE
Selector panel is shifted to the right

### ISSUE THREAD

**Issue by dimonovdd** [alexsorokoletov/Xamarin.iOS.DatePickerDialog#14]
When I took screenshots for [Auto Localization PR](https://github.com/alexsorokoletov/Xamarin.iOS.DatePickerDialog/pull/13), I noticed that this panel is shifted. I checked this behavior does not depend on this PR in any way.

![Simulator Screen Shot - iPhone SE (2nd generation) - 2021-03-09 at 23 13 48](https://user-images.githubusercontent.com/59065470/110532164-8df3ab00-812d-11eb-93cb-575e084d5d8c.png)

**Comment by alexsorokoletov** [comment]
yep, saw that too, the same problem with the original Swift library. I think the width is too small

**Comment by dimonovdd** [comment]
I'm going to do PR with a few Fixes. I will create different Issues for these fixes
